client/src/types/Flight.ts

```diff
@@ -46,6 +46,14 @@ export interface Flight {
   airbuddies?: Flight[];
 }
 
+export const enum FLIGHT_STATUS {
```

Comment:
best practice is to use union type instead of enums if it's not absolutely necessary to. use an enum: https://fettblog.eu/tidy-typescript-avoid-enums/

Reply:
I don't like to but plain strings everywhere...
I adjusted the code to a recommendation in the blog article but keep my identifiers ;-)